### PR TITLE
Support search paths in jinja2

### PR DESCRIPTION
**Pull request by pvanderlinden**
Fixes issue #473

## Proposed Changes

  - the jinja2 input now uses the search_paths when rendering
  - the jinja2_render_file jsonnet callback now uses the search_paths when rendering

### CODE REVIEW

**Comment by pvanderlinden** [comment]
@googlebot I signed it!

File: kapitan/utils.py [hunk 1]
```diff
@@ -120,7 +121,7 @@ def render_jinja2_file(name, context, jinja2_filters=defaults.DEFAULT_JINJA2_FIL
     return env.get_template(filename).render(context)
 
 
-def render_jinja2(path, context, jinja2_filters=defaults.DEFAULT_JINJA2_FILTERS_PATH):
+def render_jinja2(path, context, search_paths=None, jinja2_filters=defaults.DEFAULT_JINJA2_FILTERS_PATH):
```
Comment (adrianchifor) [inline kapitan/utils.py:124]: To keep things backwards-compatible, maybe `search_paths` as 4th param? That way we don't have to change our tests either

Comment (pvanderlinden) [reply kapitan/utils.py]: I should have done that in the first place, fixed that now.

**Comment by adrianchifor** [comment]
Thanks a lot of the PR :) LGTM

**Comment by ramaro** [comment]
LGTM, thanks!

**Comment by pvanderlinden** [comment]
> LGTM, thanks!

Thanks!

**Pull request merged**